### DISCUSSION TITLE
Setting up tenancy global scopes problem

### DISCUSSION THREAD

**Discussion by dev-federico** [filamentphp/filament#7542] [Help]
According to the documentation examples, we must add a belongsToMany relationship to the resource model (and this is ok); when we follow the security guide, in the "Using tenant-aware middleware to apply global scopes" section, is requested to do the scope appending "whereBelongsTo(Filament::getTenant())" to the query. In this way, we get an error, because whereBelongsTo Laravel Method expects a belongsTo relationship and not a belongsToMany one. Is this a BUG or are we missing something?

**Comment by danharrin** [comment]
Since `whereBelongsTo` only supports BelongsTo and not BelongsToMany, you should probably use `whereRelation` or `whereHas`

**Reply by dev-federico** [reply]
Thank you! I will try this way
